com.microsoft.mrtk.core/Utilities/CameraFOVChecker.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// Camera extension methods to test if colliders are within camera's FOV. Uses caching to improve performance and
+    /// ensure values are only computed once per frame.
+    /// </summary>
+    public static class CameraFOVChecker
+    {
+        private static int inFOVLastCalculatedFrame = -1;
+#if !NETFX_CORE
```

Comment:
We can probably get rid of this `#if` now, since .NET no longer exists for UWP, and just simplify to the `ValueTupe` code paths

Reply:
will remove :)

Reply:
done :)